### DISCUSSION TITLE
Is this compatible with the new Hero 11 in 10bit?

### DISCUSSION THREAD

**Discussion by EvaFPV** [xtremestuff/protune-transforms#4] [General]
Hiya, is this compatible with the Hero11’s 10bit color? Thanks!

**Comment by RikshaDriver** [comment]
I don't have an 11 to test with unfortunately, but assuming the Protune Native settings are aligned to previous GoPro models, it should work.

**Reply by EvaFPV** [reply]
tried it out and it seems to! i am in no way a pro so there may be things i’m missing but i got the best color grade i’ve done yet from using this so that’s something!
